packages/utxo-lib/tests/__fixtures__/compose.ts

```diff
@@ -39,7 +63,7 @@ export const composeTxFixture = [
                     amount: '100000',
                     type: 'payment',
                     customField: 'prove that payment output is generic',
-                },
+                } as AnyComposeFinalResult['outputs'][number], // hack for `customField`
```

Comment:
Why don't you declare the same thing for `Fixture.result` as for `Fixture.request` if you want to support `customField`?
Boilerplate is quicker to read than inconsistent approaches.

Reply:
Got it! Did not realise that.